react/features/base/conference/actions.js

```diff
@@ -199,7 +199,8 @@ export function conferenceFailed(conference: Object, error: string) {
         // Make the error resemble an Error instance (to the extent that
         // jitsi-meet needs it).
         error: {
-            name: error
+            name: error,
+            recoverable: undefined
```

Comment:
why do we need to be explicit about this?

Reply:
Flow was complaining about missing property and this silenced it. I can eventually try to look for another way